dags/atd_moped_data_tracker_sync.py

```diff
@@ -0,0 +1,72 @@
+# Test locally with: docker compose run --rm airflow-cli dags test atd_moped_data_tracker_sync
+
+import os
+
+from airflow.models import DAG
+from airflow.operators.docker_operator import DockerOperator
+from pendulum import datetime, duration
+
+from utils.onepassword import get_env_vars_task
+from utils.slack_operator import task_fail_slack_alert
+from utils.knack import get_date_filter_arg
+
+DEPLOYMENT_ENVIRONMENT = os.getenv("ENVIRONMENT", "development")
+
+DEFAULT_ARGS = {
+    "owner": "airflow",
+    "depends_on_past": False,
+    "start_date": datetime(2015, 1, 1, tz="America/Chicago"),
+    "email_on_failure": False,
+    "email_on_retry": False,
+    "retries": 0,
+    "retry_delay": duration(minutes=5),
+    "on_failure_callback": task_fail_slack_alert,
+}
+
+REQUIRED_SECRETS = {
+    "HASURA_ENDPOINT": {
+        "opitem": "Moped Hasura Admin",
+        "opfield": f"{DEPLOYMENT_ENVIRONMENT}.Endpoint",
+    },
+    "HASURA_ADMIN_SECRET": {
+        "opitem": "Moped Hasura Admin",
+        "opfield": f"{DEPLOYMENT_ENVIRONMENT}.Admin Secret",
+    },
+    "KNACK_DATA_TRACKER_APP_ID": {
+        "opitem": "Knack AMD Data Tracker",
+        "opfield": f"{DEPLOYMENT_ENVIRONMENT}.appId",
+    },
+    "KNACK_DATA_TRACKER_API_KEY": {
+        "opitem": "Knack AMD Data Tracker",
+        "opfield": f"{DEPLOYMENT_ENVIRONMENT}.apiKey",
+    },
+}
+
+
+with DAG(
+    dag_id="atd_moped_data_tracker_sync",
+    description="sync Moped project data to Knack Data Tracker projects table",
+    default_args=DEFAULT_ARGS,
+    schedule_interval="0 6 * * *" if DEPLOYMENT_ENVIRONMENT == "production" else None,
```

Comment:
This schedule is TBD so I'll adjust as needed.

Reply:
Considering this going to be a really light load, i was thinking we'd run it hourly.

It does seem like overkill, since 99% of the changes to projects are not going to be the pieces of data that we sync (project id, title, signal components). I will check with AMD folks 🙏

Reply:
Just made this update and we can adjust if needed in the future. 🙏